### ISSUE TITLE
http2-1.5.0 spec tests failing

### ISSUE THREAD

**Issue by juhp** [kazu-yamamoto/http2#7]
Some tests failed in the nightly stackage build:

```
> /tmp/stackage-build8/http2-1.5.0$ dist/build/spec/spec

HPACK.Decode
  fromHeaderBlock
    decodes HeaderList in request
    decodes HeaderList in response
    decodes HeaderList even if an entry is larger than DynamicTable
HPACK.Encode
  encodeHeader and decodeHeader
    works for Naive FAILED [1]
    works for NaiveH FAILED [2]
    works for Static FAILED [3]
    works for StaticH FAILED [4]
    works for Linear FAILED [5]
    works for LinearH FAILED [6]
HPACK.Huffman
  encode and decode
    duality
  encode
    encodes
  decode
    decodes
HPACK.Integer
  encode and decode
    duality
    duality
    duality
    duality
    duality
    duality
    duality
    duality
HTTP2.Frame
  encodeFrameHeader & decodeFrameHeader
    encode/decodes frames properly
  encodeFrame & decodeFrame
    encode/decodes frames properly
    encode/decodes padded frames properly
HTTP2.Priority
  priority tree
    enqueue and dequeue frames from Firefox properly
  base priority queue
    queues entries based on weight
    deletes properly
    deletes entries properly

Failures:

  test/HPACK/EncodeSpec.hs:16: 
  1) HPACK.Encode, encodeHeader and decodeHeader, works for Naive
       uncaught exception: IOException of type NoSuchThing (bench-hpack/headers.hs: openFile: does not exist (No such file or directory))

  test/HPACK/EncodeSpec.hs:18: 
  2) HPACK.Encode, encodeHeader and decodeHeader, works for NaiveH
       uncaught exception: IOException of type NoSuchThing (bench-hpack/headers.hs: openFile: does not exist (No such file or directory))

  test/HPACK/EncodeSpec.hs:20: 
  3) HPACK.Encode, encodeHeader and decodeHeader, works for Static
       uncaught exception: IOException of type NoSuchThing (bench-hpack/headers.hs: openFile: does not exist (No such file or directory))

  test/HPACK/EncodeSpec.hs:22: 
  4) HPACK.Encode, encodeHeader and decodeHeader, works for StaticH
       uncaught exception: IOException of type NoSuchThing (bench-hpack/headers.hs: openFile: does not exist (No such file or directory))

  test/HPACK/EncodeSpec.hs:24: 
  5) HPACK.Encode, encodeHeader and decodeHeader, works for Linear
       uncaught exception: IOException of type NoSuchThing (bench-hpack/headers.hs: openFile: does not exist (No such file or directory))

  test/HPACK/EncodeSpec.hs:26: 
  6) HPACK.Encode, encodeHeader and decodeHeader, works for LinearH
       uncaught exception: IOException of type NoSuchThing (bench-hpack/headers.hs: openFile: does not exist (No such file or directory))

Randomized with seed 2000255506

Finished in 0.0208 seconds
27 examples, 6 failures
```

**Comment by kazu-yamamoto** [comment]
Released v1.5.1.

**Comment by juhp** [comment]
Thank you!
